src/ovirtapi.js

```diff
@@ -371,6 +371,9 @@ OvirtApi = {
       return {
         id: c.id,
         consoleUser: c.console_user === 'true',
+        user: {
+          id: c.user.id,
```

Comment:
is `c.user !== undefined` always true?

Reply:
Yes, if user is unknown, in that case `user` gonna be:
```
"user": {
   "user_name": "Unknown"
},
```